docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
```

Comment:
In the current setup on the master branch it is assumed that Literate is part of the dependencies of the example itself but not of the documentation.

Reply:
Makes sense! Fixed.